app/Http/Requests/UserUpdateRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserUpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
instead of just returning the 'true' value let's put the policy check here and remove it from controller action:
`return $this->user()->can('update', User::find($this->route('id')));`
 or if you use model binding:
 `return $this->user()->can('update', $this->route('user'));`